services/serverSide/ticketSummary.ts

```diff
@@ -108,3 +108,12 @@ export const getTicketSentiment = async (
 
   return sentiment;
 };
+
+export const getTicketSummaryAndSentiment = async (ticketId: string) => {
+  const ticketContent = await getTicketContent(ticketId);
+
+  // eslint-disable-next-line max-len
+  const prompt = `\n\n\nNow give summary for above content in 1-2 lines. and \n\n\nNow give what is the sentiment of person tagged CONTACT in 1 line with an facial expression emoji. Format Example: Sanjay’s sentiment is slightly sad 😔`;
```

Comment:
rename prompt to instructions
and replace it with: You have to generate ticket summary and the sentiment of person tagged CONTACT in 1 line with an facial expression emoji (Format Example: Sanjay’s sentiment is slightly sad 😔).

Reply:
Done